### ISSUE TITLE
[Arcs] The comprehensive Judge's Chosen question

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#43]
[Judge's Chosen](https://cards.ledergames.com/card/ARCS-F2404) can take actions with the Judge's ships. However, the acting player is still the Judge's Chosen, not the Judge.

This means if the JC ignores/is not bound by Truce Law but the Judge is bound by Truce Law, the JC can take actions with Judge's ships that breaks the Truce Law.

This should also mean JC should suffer all the consequences of those actions
1. the JC provokes outrage for destroying a city or for [Empathy for All](https://cards.ledergames.com/card/ARCS-F1409) with Judge's ships
2. the JC loses Commonwealth membership for harming a Free city with Judge's ships

However, one agent of the Judge's is captured when the Judge's ships are moved into a [Gate Port](https://lederspotlight.netlify.app/card/ARCS-L08). This is due to the way it is phrased
> When _Rival ships take a move_ into a gate you control ...

Was this reasoning correct?

**Comment by turmoilbyrd** [comment]
This question https://github.com/LederCards/cards/issues/82 deals with the self-hits that JC may deal to the Judge's ships

**Comment by jdyearsley** [comment]
Yes, that reasoning is correct. I'll add an FAQ for Judge's Chosen and its interaction with Gate Ports.
